src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBroker.cs

```diff
@@ -46,9 +49,19 @@ public bool CanInvokeBroker(CoreUIParent uiParent)
 
             uiParent.CallerViewController.InvokeOnMainThread(() =>
             {
-                result = UIApplication.SharedApplication.CanOpenUrl(new NSUrl(BrokerParameter.BrokerV2));
+                result = UIApplication.SharedApplication.CanOpenUrl(new NSUrl(BrokerParameter.BrokerV3));
```

Comment:
`result` is an overloaded term, please use somehting with more local meaning, like `canOpenUrl`

Reply:
thanks for the suggestions...maybe it's better? also...cannot test it because my mac gave out. have to reinstall and update a bunch of stuff :(

---
In reply to: [323184596](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1383#discussion_r323184596) [](ancestors = 323184596)